### PR TITLE
add toggle action

### PR DESCRIPTION
**Pull request by jtr109**
- fix url for getting executions list
- add toggle action

### CODE REVIEW

**Comment by jtr109** [comment]
Hi, your repository is great and I am glad to use it.

I found some actions of dkron have not been implement in pydkron. So I try to do it.

It my pleasure if the PR be merged. Thx!

File: pydkron/client.py [hunk 1]
```diff
@@ -128,7 +128,25 @@ def get_executions(self, name):
         """
         Get the job executions for a named job
         """
-        resp = self._call(_GET, "/executions/%s" % name)
```
Comment (jtr109) [inline pydkron/client.py:131]: The url I found to get executions of job is `/v1/jobs/<job-name>/executions/`.

So I change both code and tests.

FYI.

Comment (jtr109) [inline pydkron/client.py:131]: The url I found to get executions of job is /v1/jobs/<job-name>/executions/.

So I change both code and tests.

FYI.

**Comment by jvineet** [comment]
@oldmantaiter any update on when this PR with fix for executions api path will be merged/resolved. We are trying to use the pydkron to export execution information for jobs running in dkron and this is causing problems. If possible, we would like to not maintain a separate fork for pydkron.

**Comment by oldmantaiter** [comment]
@jvineet @jtr109 - I no longer use dkron for work, that being said, I should not neglect this project as I have. Will review this PR to add this functionality. My sincere apologies for not paying attention to this.

File: pydkron/client.py [hunk 2]
```diff
@@ -128,7 +128,25 @@ def get_executions(self, name):
...
+        resp = self._call(_GET, "/jobs/%s/executions/" % name)
         if resp.status_code == 404:
             raise DkronJobNotFound("Job %s was not found" % name)
         return resp.json()
+
+    def toggle(self, name):
+        '''toggle job
```
Comment (oldmantaiter) [inline pydkron/client.py]: Can `toggle job` from the comment, the `Enable / disable job` comment below is enough

**Review by oldmantaiter — CHANGES_REQUESTED** [review]
@jtr109 - Can you add a test for `toggle` and squash all these changes into a single commit?

File: pydkron/client.py (hunk 2 above)
Comment (jtr109) [reply pydkron/client.py]: Got it.

**Comment by jtr109** [comment]
> @jtr109 - Can you add a test for `toggle` and squash all these changes into a single commit?

@oldmantaiter - Thanks for your reply. It is all done as you asked.

File: pydkron/client.py [hunk 3]
```diff
@@ -128,7 +128,25 @@ def get_executions(self, name):
...
+        Enable/disable a job
+
+        Arguments:
+            name {string} -- the name of job
+
+        Raises:
+            DkronJobNotFound -- raise if the job not exists
+
+        Returns:
+            DkronJob -- a job of Dkron
```
Comment (jtr109) [inline pydkron/client.py:147]: @oldmantaiter - Do you prefer the return value be a dict or a `DkronJob`?

Comment (oldmantaiter) [reply pydkron/client.py:147]: `DkronJob` is better, I'll make an issue to tackle some of the methods that return a dict and could use `DkronJob` job instead.

**Review by oldmantaiter — APPROVED** [review]
Thanks again @jtr109 - looks good to me

**Comment by jtr109** [comment]
> Thanks again @jtr109 - looks good to me

It's my pleasure. Thanks for your time.

**Pull request merged**